test/form/samples/ns-external-star-reexport/_expected/es.js

```diff
@@ -0,0 +1,11 @@
+import * as externalNs1 from 'external-ns-1';
+import * as externalNs2 from 'external-ns-2';
+
+const val = 5;
+
+var ns = /*#__PURE__*/Object.freeze(/*#__PURE__*/Object.assign(externalNs1, externalNs2, {
```

Comment:
This will be mutating `externalNs1`, I assume we should add an empty object as first parameter?

Reply:
Ah, thanks, ok I've fixed it up to use `Object.create` along with some other refinements to the various case outputs.